src/aspire/utils/numeric/pyfftw_fft.py

```diff
@@ -1,103 +1,29 @@
-from threading import Lock
+import pyfftw.interfaces.scipy_fftpack as pyfft
```

Comment:
Let's avoid non-standard module aliases. Just stick to `scipy_fftpack` or `fftpack`. Alternatively, if we have to make something new up, we can use `pyfftw_scipy`.

Reply:
Switched to class structure and new name.